### DISCUSSION TITLE
modifier la mission 46 pour quelque chose de plus réalisable

### DISCUSSION THREAD

**Discussion by ghost** [Crownicles/Crownicles#1329] [Idées acceptées / déjà implémentées]
actuellement, la mission demande de boire 3 potions sans effets, potions introuvables dans le shop en potion du jour, seulement dans les items random au milieu de tous les autres genres d'items et dans les mini-events et events si on a de la chance, ce qui fait que l'intervalle de réussite de cette mission est incertain, pouvant aller de quelques heures à très longtemps, ce qui est frustrant et bloquant. je demande un changement de cette mission par un objectif plus réalisable. 

-> Boire X potions de n'importe quelle rareté 
-> boire 1 potion sans effet

**Comment by BastLast** [comment]
go ajouter un mini event qui donne des potions sans effet :) ça peut aussi être un bon moyen de rendre cette mission plus simple.

**Comment by ghost** [comment]
c'est une solution qui peut etre faite rapidement pour etre implémentée en 3.0.2 ?

**Reply by BastLast** [reply]
ya déjà énormément de choses prévues pour la 3.0.2 mais en fonction de ce qu'on fait pour ce mini évent ça s'étudiera , version courte : aucune idée

**Reply by BastLast** [reply]
déjà c'est meme pas sur que la 3.0.2 existe

**Comment by BastLast** [comment]
anyway la demande me semble legitime du coup je crée une issue pour le mini event vu que c'est ce qui me semble le plus fun à faire.

**Comment by ghost** [comment]
ca me va aussi
